cache/filter.go

```diff
@@ -37,6 +37,21 @@ func AuthorFilter(query string) Filter {
 		return author.Match(query)
 	}
 }
+// MetadataFilter return a Filter that match a bug metadata
```

Comment:
Precise that this only work with metadata **at creation time**. It will not match metadata added later in the lifetime of the bug.

Reply:
Indeed, I've updated the comment to state that explicitly. My motivation is to be able to look up bug URLs offline, so that use-case works when metadata is limited to metadata at creation time.